test/unit/logic/transaction.js

```diff
@@ -191,6 +193,24 @@ describe('transaction', function () {
 			}).to.throw();
 		});
 
+		it('should create a transaction with data property', function () {
+			var trsData = _.cloneDeep(validTransactionData);
+			trsData.data = 'abc';
+			var trs = transaction.create(trsData);
+			expect(trs).to.be.an('object');
+			expect(trs.asset.data).to.equal('abc');
```

Comment:
Separating example setup and resulting example expectations with new line is better for readability. We do this for other tests, and should continue pattern.

```js
var trsData = _.cloneDeep(validTransactionData);
trsData.data = 'abc';
var trs = transaction.create(trsData);

expect(trs).to.be.an('object');
expect(trs.asset.data).to.equal('abc');
```